images/nginx/rootfs/build.sh

```diff
@@ -525,10 +539,13 @@ apt-get remove -y --purge \
   python \
   xz-utils \
   bc \
+  sensible-utils \
   git g++ pkgconf flex bison doxygen libyajl-dev liblmdb-dev libgeoip-dev libtool dh-autoreconf libpcre++-dev libxml2-dev
 
 apt-get autoremove -y
 
+dpkg -l | grep '^rc' | awk '{print $2}' | xargs apt-get purge --yes
```

Comment:
this command removes configuration files left after package removals.
```
apt list|grep residual

autoconf/now 2.69-11 all [residual-config]
cmake-data/now 3.13.4-1 all [residual-config]
groff-base/now 1.22.4-3 amd64 [residual-config]
libglib2.0-0/now 2.58.3-2 amd64 [residual-config]
libmagic1/now 1:5.35-4 amd64 [residual-config]
libpython2.7-minimal/now 2.7.16-2 amd64 [residual-config]
man-db/now 2.8.5-2 amd64 [residual-config]
mime-support/now 3.62 all [residual-config]
perl/now 5.28.1-6 amd64 [residual-config]
python2.7-minimal/now 2.7.16-2 amd64 [residual-config]
readline-common/now 7.0-5 all [residual-config]
```

Reply:
would be useful to make this an inline comment in the file

Reply:
done